.gitignore

```diff
@@ -4,6 +4,11 @@
 # Log file
 *.log
 testIgnore/
+Projects/
```

Comment:
Why this is needed?

Reply:
Because I want to create a new project for my homework.

Reply:
So you don't need to ignore this folder

Reply:
I understood.  I deleted the "project" folder and added the "out" folder in the gitignore file.